modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-upgrading-projects.adoc
+
+:osdk_ver: v1.8.0
+:osdk_ver_n1: v1.3.0
+
+[id="osdk-upgrading-v130-to-v180_{context}"]
+= Upgrading projects for Operator SDK {osdk_ver}
+
+The following upgrade steps must be performed to upgrade an existing Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+- Operator SDK {osdk_ver} installed
+- Operator project that was previously created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+. Make the following changes to your `PROJECT` file:
+
+.. Update the `PROJECT` file `plugins` object to use `manifests` and `scorecard` objects.
++
+The `manifests` and `scorecard` plug-ins that create Operator Lifecycle Manager (OLM) and scorecard manifests now have plug-in objects that direct `create` subcommands to create related files.
```

Comment:
nit: "that direct `create` subcommands" reads a little awkwardly to me.  "that run"?

I'm not familiar with the material, so ignore if you can tell that I don't know what I'm talking about.

Reply:
Almost changed this but didn't. And now I will (to "objects for running `create`...)"